### PR TITLE
VCTK Using Path and glob instead of walk_files 

### PR DESCRIPTION
**Pull request by krishnakalyan3**
Remove walk_files from VCTK dataset

Fixes:
#1051
#1069

### CODE REVIEW

**Comment by krishnakalyan3** [comment]
This PR was not tested as this dataset is no longer available.

File: torchaudio/datasets/vctk.py [hunk 1]
```diff
@@ -123,9 +122,7 @@ def __init__(self,
                 "with `download=True` to donwload the latest version."
             )
 
-        walker = walk_files(
-            self._path, suffix=self._ext_audio, prefix=False, remove_suffix=True
-        )
+        walker = sorted(str(p.stem) for p in Path(self._path).glob('*/*/*' + self._ext_audio))
```
Comment (mthrok) [inline torchaudio/datasets/vctk.py:125]: Can you change the pattern to `'**/*' + self._ext_audio`?

That should match the behavior of the original implementation.

**Review by mthrok — COMMENTED** [review]
Looks good. Please do the honor to delete the implementation of `walk_files`.

**Comment by mthrok** [comment]
Thanks! @krishnakalyan3

**Pull request merged**